safe_transaction_service/tokens/tests/test_models.py

```diff
@@ -157,3 +157,22 @@ def test_create_from_blockchain_with_byte_data(self, get_info: MagicMock):
         token = Token.objects.get(address="0xBB9bc244D798123fDe783fCc1C72d3Bb8C189413")
         self.assertEqual(token.name, "�������������������������������\x01")
         self.assertEqual(token.symbol, "�������������������������������\x01")
+
+    @mock.patch.object(
+        Erc20Manager,
+        "get_info",
+        autospec=True,
+        return_value=Erc20Info(
+            name="Build\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00",
```

Comment:
if this should be handled like a c-type memory for strings, then 0-bytes terminate the string. Therefore I would not replace them but look for the first 0-byte and cut of everything afterwards

Reply:
Yeah, but are we sure `0-bytes` are not allowed anywhere else?